### PR TITLE
add erlang/elixir source to data

### PR DESCRIPTION
**Pull request by tsloughter**

### CODE REVIEW

**Comment by tsloughter** [comment]
Well that isn't right. I Checked the preview and it has the bar full... Is that because there isn't a release yet?

**Comment by austinlparker** [comment]
Lemme take a look, maybe it’s a bug in how the data gets generated

**Comment by tsloughter** [comment]
I created a release so it might work now if it gets rerun/rebuilt.

**Comment by tsloughter** [comment]
Argh, I created a release yesterday https://github.com/open-telemetry/opentelemetry-erlang-api/releases and then this morning force pushed to get a new build and it is still just a solid black bar...

**Comment by austinlparker** [comment]
oh! ok, I think I know what's going on here - there's probably no style information in the css for this.

File: data/progress_source.yaml [hunk 1]
```diff
@@ -74,6 +74,15 @@ data:
     releaseName: "n/a"
     releaseDescription: "n/a"
     releaseUrl: ""
+  - label: Erlang/Elixir
```
Comment (austinlparker) [inline data/progress_source.yaml]: If you change this value to 'Erlang' it'll work - the slash in the label breaks the css class.

Comment (tsloughter) [reply data/progress_source.yaml]: Ah, *facepalm*, ok, I've pushed a change.

**Comment by tsloughter** [comment]
The full black bar is fixed but all of them are now showing no releases. Maybe that is just a preview thing?

**Comment by austinlparker** [comment]
Yeah, I don’t think we run the script to pull milestones on PRs

**Pull request merged**